gallery/src/components/NxDescriptionList/NxDescriptionListLinkExample.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxDescriptionList } from '@sonatype/react-shared-components';
+
+const NxDescriptionListLinkExample = () =>
+  <NxDescriptionList>
+    <NxDescriptionList.LinkItem href="#/"
+                                term="This is the NxDescriptionList.LinkItem component"
+                                description="Clicking this row will give you an alert message." />
```

Comment:
Minor, but this doesn't give you an alert in the Link Example

Reply:
[db5788c](https://github.com/sonatype/sonatype-react-shared-components/pull/752/commits/db5788c31295be3cfab9fce5e63880f31876bf4d)